litellm/llms/ollama_chat.py

```diff
@@ -300,7 +300,7 @@ def get_ollama_response(
         model_response["choices"][0]["message"] = message
         model_response["choices"][0]["finish_reason"] = "tool_calls"
     else:
-        model_response["choices"][0]["message"] = response_json["message"]
+        model_response["choices"][0]["message"]["content"] = response_json["message"]["content"]
```

Comment:
this doesn't fix the problem of the message being a dict object instead of pydantic.

@paneru-rajan can you share a test you wrote to see if this fixed the problem, and that it's passing?

Reply:
I tested, `model_response["choices"][0]["message"]` is a pydantic Message Object.

As before we were **replacing** it with dict, which is what was the issue. Since i am just assigning the content which is a string, which does not override the existing Message Object.

<img width="1140" alt="image" src="https://github.com/BerriAI/litellm/assets/4735661/c78bfbb1-42ce-472f-b5e8-b5de4ef7ccb3">

Reply:
Following image shows assert doesn't raise an exception.

<img width="1286" alt="image" src="https://github.com/BerriAI/litellm/assets/4735661/d24cacac-8065-418c-97c6-72c7fa5cda14">